service_container/expression_language.rst

```diff
@@ -67,7 +67,7 @@ to another service: ``App\Mailer``. One way to do this is with an expression:
             $services->set(MailerConfiguration::class);
 
             $services->set(Mailer::class)
-                ->args([expr("service('App\\Mail\\MailerConfiguration').getMailerMethod()")]);
+                ->args([expr("service('App\\\\Mail\\\\MailerConfiguration').getMailerMethod()")]);
```

Comment:
The result is so ugly and confusing, that we might think of adding a comment explaining this. Example:

```php
$services->set(Mailer::class)
    // because of the escaping applied by PHP, you must add 4 backslashes for each original backslash
    ->args([expr("service('App\\\\Mail\\\\MailerConfiguration').getMailerMethod()")]);
```